File: application/src/test/java/com/smtm/application/categories/TestCategoriesConfiguration.java
package com.smtm.application.categories;

import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.Configuration;
import com.smtm.application.categories.v1.CategoriesController;

@Configuration
public class TestCategoriesConfiguration {

    @Bean
    public CategoriesController categoriesController() {
        return new CategoriesController();
    }
}

File: application/src/test/kotlin/com/smtm/application/categories/TestCategoriesConfiguration.kt
package com.smtm.application.categories

import com.smtm.application.categories.v1.CategoriesController
import org.springframework.context.annotation.Bean
import org.springframework.context.annotation.Configuration

@Configuration
class TestCategoriesConfiguration {

    @Bean
    fun categoriesController(): CategoriesController {
        return CategoriesController()
    }
}
